src/main/java/io/dashboardhub/pipelinedashboard/domain/User.java

```diff
@@ -19,14 +20,15 @@
     private String uid;
 
     @NotNull
-    @Size(min = 1, max = 255)
     private String username;
 
+    @Size(min = 5, max = 255)
     private String name;
 
-//    @ManyToMany
-//    @JoinTable(name="users_projects")
-//    private Set<Project> projects;
+    @Email
+    private String email;
+
+    private String lastLoggedIn;
```

Comment:
Yup. All lands in my mailbox. I get to discover interesting projects pretty much every other day though. Like this one, which I think I even recognize from HN :).